package.json

```diff
@@ -42,5 +51,704 @@
+    "timed-out": "^4.0.1",
+    "tmp": "^0.0.33",
+    "to-readable-stream": "^1.0.0",
+    "to-regex-range": "^5.0.1",
+    "tough-cookie": "^2.5.0",
+    "tr46": "^0.0.3",
+    "true-case-path": "^2.2.1",
+    "toidentifier": "^1.0.1",
+    "ts-command-line-args": "^2.2.1",
+    "ts-essentials": "^7.0.3",
+    "tsort": "^0.0.1",
+    "tsconfig-paths": "^3.12.0",
+    "ts-generator": "^0.1.1",
+    "tslib": "^1.14.1",
+    "tunnel-agent": "^0.6.0",
+    "tsutils": "^3.21.0",
+    "through": "^2.3.8",
+    "tweetnacl-util": "^0.15.1",
+    "type-check": "^0.4.0",
+    "tweetnacl": "^0.14.5",
+    "type-detect": "^4.0.8",
+    "type-is": "^1.6.18",
+    "type-fest": "^0.20.2",
+    "typical": "^2.6.1",
+    "typedarray-to-buffer": "^3.1.5",
+    "u2f-api": "^0.2.7",
+    "ultron": "^1.1.1",
+    "uglify-js": "^3.14.5",
+    "type": "^1.2.0",
+    "universalify": "^0.1.2",
+    "url": "^0.11.0",
+    "url-parse-lax": "^3.0.0",
+    "url-set-query": "^1.0.0",
+    "url-to-options": "^1.0.1",
+    "unbox-primitive": "^1.0.1",
+    "utf8": "^3.0.0",
+    "uri-js": "^4.4.1",
+    "util-deprecate": "^1.0.2",
+    "utf-8-validate": "^5.0.8",
+    "unpipe": "^1.0.0",
+    "utils-merge": "^1.0.1",
+    "usb": "^1.9.2",
+    "validate-npm-package-license": "^3.0.4",
+    "util": "^0.12.4",
+    "vary": "^1.1.2",
+    "verror": "^1.10.0",
+    "varint": "^5.0.2",
+    "uuid": "^3.4.0",
+    "web3": "^1.5.3",
+    "v8-compile-cache": "^2.3.0",
+    "web3-core": "^1.5.3",
+    "web3-bzz": "^1.5.3",
+    "web3-core-promievent": "^1.5.3",
+    "web3-core-requestmanager": "^1.5.3",
+    "web3-core-subscriptions": "^1.5.3",
+    "web3-core-helpers": "^1.5.3",
+    "web3-eth": "^1.5.3",
+    "web3-core-method": "^1.5.3",
+    "web3-eth-abi": "^1.5.3",
+    "web3-eth-accounts": "^1.5.3",
+    "web3-eth-contract": "^1.5.3",
+    "web3-eth-personal": "^1.5.3",
+    "web3-net": "^1.5.3",
+    "web3-eth-iban": "^1.5.3",
+    "web3-providers-ipc": "^1.5.3",
+    "web3-providers-ws": "^1.5.3",
+    "web3-providers-http": "^1.5.3",
+    "web3-shh": "^1.5.3",
+    "web3-utils": "^1.6.1",
+    "websocket": "^1.0.34",
+    "web3-eth-ens": "^1.5.3",
+    "webidl-conversions": "^3.0.1",
+    "whatwg-url": "^5.0.0",
+    "which": "^2.0.2",
+    "which-module": "^1.0.0",
+    "which-boxed-primitive": "^1.0.2",
+    "which-pm-runs": "^1.1.0",
+    "wide-align": "^1.1.3",
+    "window-size": "^0.2.0",
+    "wordwrap": "^1.0.0",
+    "wrap-ansi": "^2.1.0",
+    "wrappy": "^1.0.2",
+    "which-typed-array": "^1.1.7",
+    "word-wrap": "^1.2.3",
+    "write": "^1.0.3",
+    "wordwrapjs": "^4.0.1",
+    "ws": "^7.4.6",
+    "xtend": "^4.0.2",
+    "xhr-request": "^1.1.0",
+    "xhr2-cookies": "^1.1.0",
+    "xhr": "^2.6.0",
+    "xmlhttprequest": "^1.8.0",
+    "xhr-request-promise": "^0.1.3",
+    "yallist": "^4.0.0",
+    "yaeti": "^0.0.6",
+    "yargs-parser": "^2.4.1",
+    "y18n": "^3.2.2",
+    "yargs": "^4.8.1",
+    "yargs-unparser": "^1.6.0",
+    "yn": "^3.1.1"
+  },
```

Comment:
WHAT ARE THOOOSE!

Reply:
lololol funny af. That was created by the npm init

Reply:
Weird.

These are definitely not needed/required/wanted to be here though right?